### PR TITLE
Slim down content, create appendix

### PR DESCRIPTION
**Pull request by sparkwebdev**
Slim down Lesson zero; remove Docker setup (dev environment tutorial ……to be defined); move supplementary content into an appendix

### CODE REVIEW

**Comment by carmemias** [comment]
All looks good to me. 
As mentioned during our chat earlier, perhaps the `What do WordPress developers do?` section is not needed? As you said, it will be fairly obvious when doing the module...

File: lesson0.md [hunk 1]
```diff
@@ -1,155 +1,55 @@
...
+- [Resources](#resources)
 
-WordPress is an Open Source Content Management System (CMS), released in 2003 by Matt Mullenweg and Mike Little. As of 2017, it has 59% of the CMS marketshare and 28% of all websites, worldwide.
+---
 
-Although WordPress started as a blogging platform, it can also be used for building websites (large and small), e-commerce sites (with the WooCommerce plugin), private community platforms (with the BuddyPress plugin), forums (with the bbPress plugin), etc.
+## What is WordPress?
 
-Some examples of these could be: [Kinning Park Complex](http://www.kinningparkcomplex.org/), [Sony Music](https://www.sonymusic.com/), [Duracell Lighting](http://www.duracelllighting.com/), and [The Rolling Stones](http://www.rollingstones.com/)
+WordPress is an Open Source Content Management System (CMS), released in 2003 by Matt Mullenweg and Mike Little. It was released in 2003 by Matt Mullenweg and Mike Little. As of 2017, it has 59% of the CMS marketshare and 28% of all websites, worldwide. It is:
```
Comment (carmemias) [inline lesson0.md]: `released in 2003 by MW and ML` is mentioned twice.
Also, we could update the stats to the 2018 numbers, i.e. 30%

File: lesson0.md [hunk 2]
```diff
@@ -1,155 +1,55 @@
...
-
-*   You can build and maintain websites for clients using, and adapting, existing WordPress **Themes** ( = collection of templates for the frontend ) and **Plugins** ( = extensions to the default WordPress functionality ).
-*   You can create WordPress Themes for clients or to sell them on platforms such as the [Theme Directory](https://en-gb.wordpress.org/themes/) or the [ThemeForest](https://themeforest.net).
-*   You can create WordPress Plugins for clients or to sell on the [Plugin Directory](https://en-gb.wordpress.org/plugins/).
-
-Of course, you can also earn a living as a content editor, a blogger or a business owner with a WordPress website. But that topic is outside the scope of this course ;-) .
-
-We will cover Themes and Plugins extensively during the course but, in the meantime, if you want to find out more, please refer to the following links in the WordPress Codex:
+> **Exercise:** Create a blog with WordPress.com and become familiar with the key WordPress concepts. If you get stuck with any of the steps, dont worry, check the [WordPress.com support site](https://en.support.wordpress.com/) or post a question on Slack.
```
Comment (carmemias) [inline lesson0.md]: Spelling `dont worry`should be `don't worry`

File: lesson0.md [hunk 3]
```diff
@@ -1,155 +1,55 @@
...
-
-If you wish to find out more about Free Software, Open Source and licensing, please watch the excellent documentary ["Revolution OS"](https://www.youtube.com/watch?v=4vW62KqKJ5A) on YouTube.
-
-
-These sections have been sourced from:
-* [GNU - Free Software Definition](https://www.gnu.org/philosophy/free-sw.html)
-* [GNU - What is Copyleft?](https://www.gnu.org/copyleft/copyleft.html)
-* [Make WordPress Training - What is Open Source](https://make.wordpress.org/training/handbook/user-lessons/what-is-open-source/)
-* [Opensource.com - What's the difference between open source software and free software?](https://opensource.com/article/17/11/open-source-or-free-software)
+The WordPress module is four weeks of classes designed to get you up and running with WordPress and learn it's various features so that you can work with your fellow students to build a WordPress site. You will use use your existing HTML, CSS and JavaScript knowledge to create a WordPress theme and plugin and we will explore the WordPress Rest API to write a small React web app.
```
Comment (petersuhm) [inline lesson0.md]: * REST instead of Rest

**Comment by petersuhm** [comment]
Looks good, @sparkwebdev! :)

**Pull request merged**